openshift/release/create-release-branch.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Copyright 2019 The OpenShift Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: create-release-branch.sh v0.4.1 release-0.4
+
+set -ex # Exit immediately on error.
+
+source openshift/release/common.sh
+
+release=$1
+target=$2
+
+# Fetch the latest tags and checkout a new branch from the wanted tag.
+git fetch upstream -v --tags
+git checkout -b "$target" "$release"
+
+# Remove GH Action hooks from upstream
+rm -rf .github/workflows
```

Comment:
I would not remove all of them.

Reply:
I would keep unit/integration tests.

Reply:
I removed the `rm` completely, as I saw you remove the unneeded tests in [remove-workflow-tests.s](https://github.com/openshift-knative/kn-plugin-func/blob/main/openshift/scripts/remove-workflow-tests.sh)  anyhow (which gets called in [apply-midstream-patches.sh](https://github.com/creydr/knative-func/blob/7ff10a6c26d07b8ccc4745bd61bf2ecc628c0907/openshift/release/apply-midstream-patches.sh#L33))